libsolidity/ast/Types.cpp

```diff
@@ -536,8 +536,7 @@ bool FixedPointType::isImplicitlyConvertibleTo(Type const& _convertTo) const
 bool FixedPointType::isExplicitlyConvertibleTo(Type const& _convertTo) const
 {
 	return _convertTo.category() == category() ||
-		_convertTo.category() == Category::Integer ||
-		_convertTo.category() == Category::FixedBytes;
+		_convertTo.category() == Category::Integer;
```

Comment:
Hmm, should also have another check that it is not an address.

Reply:
Done.